manifests/examples/kvm/stretch-virtio-partitions.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: debian-{system.release}-{system.architecture}-{%Y}{%m}{%d}
+provider:
+  name: kvm
+  virtio_modules:
+  - virtio_pci
+  - virtio_blk
+bootstrapper:
+  workspace: /target
+system:
+  release: stretch
+  architecture: amd64
+  bootloader: grub
+  charmap: UTF-8
+  locale: en_US
+  timezone: UTC
+volume:
+  backing: raw
+  partitions:
+    type: gpt
+    boot:
+      filesystem: ext2
+      size: 1GiB
+    swap:
+      size: 128MiB
+    root:
+      filesystem: ext4
+      size: 8GiB
+    tmp:
+      mountopts:
+        - nodev
+        - noexec
+        - nosuid
+        - journal_ioprio=3
+      filesystem: ext4
+      size: 1GiB
+    var:
+      filesystem: ext4
+      size: 1GiB
+    var/tmp:
```

Comment:
I love the simplicity of how to specify the mountpaths, great idea just using the keys.
Do you think it would be possible to include the slash at the beginning of the path? It would make it a lot easier for people to understand what is going on at a quick glance.  
If you see `var` in that list you might think "hm, ok, I guess there are some special values", if however you see `/var` you just think "aha, paths, ok cool, I know how to fiddle with that..."

In the same vein we should probably rename `root` to `/` and `boot` to `/boot`, but I think this is plenty for one PR :-)